src/index.ts

```diff
@@ -1,10 +1,11 @@
-import { Users } from './services/users';
 import { Logger } from './services/logger';
 
-import type { User, ApiConfig } from './types';
+import type {User} from './types';
+import {createIoCContainer, ioc} from "./ioc";
 
-const renderUsers = async (config: ApiConfig) => {
-  const usersService = new Users(config);
+const renderUsers = async () => {
+
+  const usersService = createIoCContainer().resolve('users');
```

Comment:
Hey, why did you create one more IOC container ? 
you already imported the ioc container, so please use it `ioc.resolve('users');`